<br><br>
# <center>STATISTICAL METHODS IN FINANCE</center>
# <center>HW2</center>
<center>Fan Yang</center>
<center>UNI: fy2232</center>
<center>02/07/2018</center>

# Problem 1 $_{_{(\text{Page 491 Exec 1})}}$

***Suppose that the daily log returns on a stock are independent and normally distributed with mean 0.001 and standard deviation 0.015. Suppose you buy \$1,000 worth of this stock.<br>***
>***(a)*** *What is the probability that after one trading day your investment is worth less than \$990? (Note: The R function `pnorm()` will compute a normal CDF, so, for example, `pnorm(0.3, mean = 0.1, sd = 0.2)` is the normal CDF with mean 0.1 and standard deviation 0.2 evaluated at 0.3.)*<br>
***(b)*** *What is the probability that after five trading days your investment is worth less than $990?*

## (a). 
\begin{split}
\tilde{R}_t&=\log(1+R_t)=\log(1+\frac{990-1000}{1000})\\
&=\log 0.99
\end{split}

In [15]:
pnorm(log(0.99),0.001,0.015)

[1] 0.2306557

The probability that after one trading day your investment is worth less than \$990 is 0.23065573.

## (b). 
\begin{split}
R_t(5)&=1+\frac{990-1000}{1000}=0.99\\
1+R_t(5)&=\exp\{\tilde{R}_t+\tilde{R}_{t-1}+\tilde{R}_{t-2}+\tilde{R}_{t-3}+\tilde{R}_{t-4}\}\\
\tilde{R}_t+\tilde{R}_{t-1}+\tilde{R}_{t-2}+\tilde{R}_{t-3}+\tilde{R}_{t-4} &= \log(1+R_t(5))=\log(0.99)\\
\end{split}
Since the daily log returns on a stock are independent and normally distributed. So, $\sum_{i=0}^4\tilde{R}_{t-i} \sim N(0.001\times5,0.015\times\sqrt{5})$

In [3]:
pnorm(log(0.99),0.001*5,0.015*sqrt(5))

[1] 0.3268189

The probability that after five trading day your investment is worth less than \$990 is 0.326818876.

# Problem 2 $_{_{(\text{Page 492 Exec 2})}}$

***The yearly log returns on a stock are normally distributed with mean 0.08 and standard deviation 0.15. The stock is selling at \$80 today. What is the probability that 2 years from now it is selling at \$90 or more?<br>***

\begin{split}
R_t(2)&=1+\frac{90-80}{80}=\frac{9}{8}\\
1+R_t(2)&=\exp\{\tilde{R}_t+\tilde{R}_{t-1}\}\\
\tilde{R}_t+\tilde{R}_{t-1} &= \log(1+R_t(2))=\log(\frac{9}{8})\\
\end{split}
Since the daily log returns on a stock are independent and normally distributed. So, $\sum_{i=0}^1\tilde{R}_{t-i} \sim N(0.08\times2,0.15\times\sqrt{2})$

In [45]:
1-pnorm(log(9/8),0.08*2,0.15*sqrt(2))

[1] 0.5788736

The probability that before 2 trading years your investment is worth at \$90 is 0.57887.

# Problem 3 $_{_{(\text{Page 492 Exec 3})}}$

***The daily log returns on a stock are normally distributed with mean 0.0002
and standard deviation 0.03. The stock price is now \$97. What is the probability that it will exceed \$100 after 20 trading days?<br>***

\begin{split}
R_t(20)&=1+\frac{100-97}{97}=\frac{100}{97}\\
1+R_t(20)&=\exp\{\sum_{i=0}^{19}\tilde{R}_{t-i}\}\\
\sum_{i=0}^{19}\tilde{R}_{t-i} &= \log(1+R_t(20))=\log(\frac{100}{97})\\
\end{split}
Since the daily log returns on a stock are independent and normally distributed. So, $\sum_{i=0}^{19}\tilde{R}_{t-i} \sim N(0.0002\times20,0.03\times\sqrt{20})$

In [15]:
1-pnorm(log(100/97),0.0002*20,0.03*sqrt(20))

[1] 0.4218295

The probability that before 20 trading years your investment exceed \$100 is 0.4218295.

# Problem 4 $_{_{(\text{Page 492 Exec 4})}}$

***Suppose that the forward rate is r(t) = 0.028 + 0.00042t.
<br>***
>(a) What is the yield to maturity of a bond maturing in 20 years?<br>
(b) What is the price of a par $1,000 zero-coupon bond maturing in 15 years?

## (a).
\begin{split}
y(t)&=\frac{1}{t}\int_0^tr(s)ds\\
&=\frac{1}{20}\int_0^{20}~0.028+0.00042s~ds\\
&=\frac{0.644}{20}\\
&=0.0322\\
\end{split}

## (b).
\begin{split}
P(t)&=PAR\times e^{-\int_0^tr(s)ds}\\
&=1000\times e^{-\int_0^{15}~0.028+0.00042s~ds}\\
&=1000\times e^{-0.46725}\\
&=626.723\\
\end{split}

# Problem 5 $_{_{(\text{Page 492 Exec 6})}}$

***A coupon bond has a coupon rate of 3% and a current yield of 2.8%.
<br>***
>(a) Is the bond selling above or below par? Why or why not?<br>
(b) Is the yield to maturity above or below 2.8%? Why or why not?

## (a).
\begin{split}
C&=\text{PAR}\times \text{coupon rate}=\text{PAR}\times0.03\\
C&=\text{Price}\times \text{current yield}=\text{Price}\times0.028\\
\text{so, Price}&>\text{PAR}\\
\end{split}
The bond selling is above par.

## (b).
<!---
\begin{split}
\text{YTM}&=\frac{1}{T}\log\left(\frac{\text{PAR}}{\text{Price}}\right)\\
&=\frac{1}{T}\log\left(\frac{\text{current yield}}{\text{coupon rate}}\right)\\
\frac{\text{YTM}}{\text{current yield}}&=\frac{1}{T\times\text{current yield}}\log\left(\frac{\text{current yield}}{\text{coupon rate}}\right)\\
&=\log\left(\frac{\text{current yield}}{\text{coupon rate}}\right)^{\frac{1}{T\times\text{current yield}}}\\
\end{split}
$\frac{\text{current yield}}{\text{coupon rate}}<1$, then $\left(\frac{\text{current yield}}{\text{coupon rate}}\right)^{\frac{1}{T\times\text{current yield}}}<1$
The bond selling is below par.
-->
The bond is selling above par value, then the coupon rate is greater than the current yield because the bond sells above par value, and the current yield is greater than the yield to maturity because the yield to maturity accounts for the loss of capital when at the maturity date you get back only the par value, not the entire investment. In summary,<br>
<center>coupon rate > current yield > yield to maturity.